cosmosdb/common/ps-account-failover-priority-update.ps1

```diff
@@ -3,12 +3,18 @@
 # Updating location with failoverPriority = 0 will trigger a failover
 $resourceGroupName = "myResourceGroup"
 $accountName = "mycosmosaccount"
+$resourceType = "Microsoft.DocumentDb/databaseAccounts"
+$apiVersion = "2015-04-08"
 
-$failoverPolicies = @(
+$failoverRegions = @(
     @{ "locationName"="East US"; "failoverPriority"=0 },
     @{ "locationName"="West US"; "failoverPriority"=1 }
 )
 
+$failoverPolicies = @{ 
+    "failoverPolicies"= $failoverRegions
+}
+
 Invoke-AzResourceAction -Action failoverPriorityChange `
```

Comment:
Even for a DB already configured for failover policies for the regions specified as the failover regions, this command fails:

```
Invoke-AzResourceAction : {"code":"BadRequest","message":"Failover priorities must be unique and 0 <= priority < (number of 
failover policies)\r\nActivityId: 206240be-24ec-47df-8d54-00ec7a3afc44, Microsoft.Azure.Documents.Common/2.4.0.0"}      
At C:\Users\sttramer\src\azure-docs-powershell-samples\cosmosdb\common\ps-account-failover-priority-update.ps1:18 char:1
+ Invoke-AzResourceAction -Action failoverPriorityChange `
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Invoke-AzResourceAction], ErrorResponseMessageException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.InvokAzureResourceActionCmdlet
```

Please clearly explain the prerequisities required for this script to succeed and make any necessary changes to the command.

Reply:
Not sure what you're saying here or why your script failed. Tested this again and it works properly.

Reply:
please review.

Reply:
This might be due to the region used here, which may have problems with some internal accounts - I'm picking this back up for re-review.